shared/domain/src/main/java/gov/va/vro/model/bip/BipFileUploadResp.java

```diff
@@ -11,6 +12,7 @@
  * @author warren @Date 12/7/22
  */
 @RequiredArgsConstructor
+@AllArgsConstructor
```

Comment:
The constructor annotations are not needed

Reply:
AllargsConstructor is used. Without it, error shows up. Maybe @RequiredArgsContructor is not needed.